airflow/contrib/executors/mesos_executor.py

```diff
@@ -49,7 +45,7 @@ def get_framework_name():
 
 # AirflowMesosScheduler, implements Mesos Scheduler interface
 # To schedule airflow jobs on mesos
-class AirflowMesosScheduler(mesos.interface.Scheduler, LoggingMixin):
+class AirflowMesosScheduler(mesos.interface.Scheduler):
```

Comment:
I think @ashb has asked about deprecating this executor.

Reply:
ye indeed, but for now this did fix the docs